packages/plugins/plugin-idfa/src/IdfaPlugin.tsx

```diff
@@ -3,36 +3,64 @@ import {
   PluginType,
   SegmentClient,
 } from '@segment/analytics-react-native';
-import { IdfaEvents } from './IdfaEvents';
 import type { IdfaData } from './types';
 import { AnalyticsReactNativePluginIdfa } from './AnalyticsReactNativePluginIdfa';
 
 const { getTrackingAuthorizationStatus } = AnalyticsReactNativePluginIdfa;
 
 export class IdfaPlugin extends Plugin {
   type = PluginType.enrichment;
+  private isEnabled?: boolean = false;
+  private isDisabled?: boolean;
```

Comment:
This is very confusing. I don't think also `isEnabled` needs to be an optional if it is always initialized to `false`

I think it would make more sense first by making them all positive boolean (`isEnabled`) instead of negative ones (`isDisabled`). Then renaming them:

- isDisabled -> shouldAskPermissions
- isEnabled -> isPermissionEnabled

That would make it easier to understand what each one does.